packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/contact-section/contact-section.ejs

```diff
@@ -0,0 +1,117 @@
+<%_ if (profile.relevantLinks && profile.relevantLinks.length > 0 || profile.phoneNumbers && profile.phoneNumbers.length > 0 ||profile.emails && profile.emails.length > 0 || profile.city && profile.country ) { -%>
+  
+  <h2>Contact</h2>
+      <div class="aside__container">     
+       <%_ for (const number of profile?.phoneNumbers) { -%>       
```

Comment:
Two thinks:
- What happen here if we have relevantLinks por example, but not phoneNumbers?
- profile.phoneNumbers is an array, number is an item in the array

You change this, if we have phoneNumbers we display phones